applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.c

```diff
@@ -209,6 +211,7 @@ static void bt_enabled_cb(int err)
 
 static int bonding_clear_check(void)
 {
+#if CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP
```

Comment:
Since we are not going to provide link feature, probably we could remove this part ?

Reply:
We need this here to be able to compile for nrf5340dk, this will also benefit OCT-2681